psalm-baseline.xml

```diff
@@ -797,27 +779,22 @@
     </MissingReturnType>
   </file>
   <file src="src/Generator/TypeGenerator.php">
-    <InvalidScalarArgument occurrences="4">
-      <code>$trimmedNullable</code>
-      <code>$trimmedType</code>
-      <code>$trimmedType</code>
-      <code>$trimmedType</code>
-    </InvalidScalarArgument>
-    <MissingReturnType occurrences="1">
-      <code>generate</code>
-    </MissingReturnType>
-    <MixedArgument occurrences="1">
-      <code>$this-&gt;generate()</code>
-    </MixedArgument>
-    <PossiblyInvalidPropertyAssignmentValue occurrences="2">
-      <code>$nullable</code>
-      <code>$trimmedType</code>
-    </PossiblyInvalidPropertyAssignmentValue>
-    <PropertyNotSetInConstructor occurrences="3">
-      <code>$isInternalPhpType</code>
-      <code>$nullable</code>
-      <code>$type</code>
-    </PropertyNotSetInConstructor>
+    <ImpureMethodCall occurrences="7">
+      <code>allowsNull</code>
```

Comment:
Purity of `Reflection*#get*` APIs is not properly documented within psalm: potentially necessary to have upstream improvements, but nothing to fix here